### ISSUE TITLE
Release version 1.x with Rails 5 compatibility

### ISSUE THREAD

**Issue by itay-grudev** [tcocca/acts_as_follower#98]
Could you please release a new major `1.x` version with the Rails 5 compatibility. Acquiring the gem from git is definitely not the best practice in production environments and often causes problems.

**Comment by itay-grudev** [comment]
This is currently blocking my deployment process. I can volunteer any help if required. If you don't want to create a new version, could you please inform me so I can clone your code and make my own gem.

**Comment by digitalmoksha** [comment]
Any hope of getting a little love on this gem?  I agree with @itay-grudev , requiring pulling master doesn't work when you have act_as_follower in a gem spec.

**Comment by itay-grudev** [comment]
@digitalmoksha I created a clone [acts_as_follower1](https://rubygems.org/gems/acts_as_follower1) which you can use in the meantime. But I will yank it when they release the Rails 5 functionality.

**Comment by digitalmoksha** [comment]
@itay-grudev Thanks.  I think i'm going to try [partisan](https://github.com/mirego/partisan) and see how that goes.

**Comment by Ajaxy** [comment]
@digitalmoksha did you like that `partisan`?

**Comment by digitalmoksha** [comment]
@Ajaxy I converted to it with no problems.  Of course I'm not using some of the more esoteric features of `acts_as_follower`, so it was a simple conversion.  But it suits my needs.

**Comment by Ajaxy** [comment]
Yes, for me it was good as well. Hope this one will not be abandoned, though.

**Comment by vinc** [comment]
A version of this gem compatible with rails 5 would indeed be a good thing.
